lib/response.js

```diff
@@ -248,6 +248,18 @@ res.json = function json(obj) {
   var spaces = app.get('json spaces');
   var body = stringify(val, replacer, spaces);
 
+  // escape characters that can cause old browsers to mistake json for html
+  if (app.get('json escape')) {
+      body = body.replace(/[<>&]/g, function (match) {
```

Comment:
And also since the function close not close over anything, please hoist that as well.